_component_design/popovers.md

```diff
@@ -8,38 +8,38 @@ usage: >
   Popovers present a small group of information or inputs that is presented in a panel. To preserve the user’s context, the panel is overlaid on top of the original page.
 preview-image: preview-images/popover.svg
 resource: true
-status: stable
+status: in-progress
```

Comment:
Why was the `status` updated on this element? Should be stable. See the status page to understand how this data is rendered and the explanation of each status. Look at your local build or see the screenshot earlier in this review.

Reply:
I have no idea why metadata value have changed or disappeared. Changed back to 'stable'.